test/test_search.py

```diff
@@ -6,9 +6,11 @@
 from tinygrad.device import Device, Buffer
 from tinygrad.ops import LoadOps
 from tinygrad.tensor import Tensor
+from tinygrad.helpers import getenv
 
 class TestTimeLinearizer(unittest.TestCase):
   def test_reasonable_time(self):
+    if getenv("PTX"): self.skipTest('PTX expects UOpsGraph, this passes UOps as list')
```

Comment:
Can this just be fixed instead of adding a special case? It should always be UOpsGraph I think

Reply:
removed the line and the test no longer crashes, probably got fixed in a different pr

Reply:
yea was fixed in #3786